### ISSUE TITLE
Accordion's expanded content not read by JAWS | AVT 3

### ISSUE THREAD

**Issue by mekaufman** [carbon-design-system/carbon#3831]
## Environment
 Windows 10

> Browser
Chrome

> Automated testing tool and ruleset
JAWS 

> Assistive technology used to verify
JAWS

## Detailed description

> What version of the Carbon Design System are you using?
I tested 9 and 10 from the examples on your site and got the same result i.e. http://react.carbondesignsystem.com/?path=/story/accordion--default

> What did you expect to happen?
I expected to expand the accordion element and the text within that element to be read out to the screen reader whether it was automatically next or via tab order.

> What happened instead?
I am unable to hear the text read out in the section. I can't tab to it, use arrows, or any shortcut keystrokes that I can find to get the text to be acknowledged and read out.

> What WCAG 2.1 checkpoint does the issue violate?
I'm not sure of the checkpoint, but could dig it up. Really it's just essential functionality of being able to read text on a page.

## Steps to reproduce the issue

1. Go to http://react.carbondesignsystem.com/?path=/story/accordion--default
2. Open JAWS screen reader.
3. Tab to and expand the accordion (any of them)
4. Listen to the fact that no text is read out, then try tabbing and notice it tabs to the next accordion element.

**Comment by dakahn** [comment]
Seeing the same results when tested with NVDA latest on Windows 10 in Chrome.

**Comment by dakahn** [comment]
@snidersd should this be part of the Carbon Copy Milestone?

**Comment by snidersd** [comment]
@dakahn Plz add this issue to CC milestone. Also note,  this looks like a regression. We found no issues when previously tested. Thx!
Update: Looks like this issue is unique to E11. I tested this issue with JAWS on Win 10 with Edge as well as Firefox and I can not reproduce the issue, so no need to add it to the CC Milestone. Also note, keyboard interaction is consistent with the [ARIA Authoring Practices for Accordion](https://www.w3.org/TR/wai-aria-practices/#accordion).

**Comment by dakahn** [comment]
@mekaufman could you talk a little the versions of Chrome/JAWS you're testing with?

**Comment by mekaufman** [comment]
@dakahn JAWS 19, Chrome 76 on Windows 7

**Comment by dakahn** [comment]
I'm lowering the priority here since it seems to be a JAWS 19 on Windows 7 issue maybe? We also can't replicate in Win 10. We'll leave open for reference.

**Comment by mekaufman** [comment]
JAWS 20/Windows 10/Chrome I am finally able to get the content within the accordion to read out.

**Comment by ashwinishejul** [comment]
[mekaufman](https://github.com/mekaufman) can you please provide the code
